### PR TITLE
refactor(trackerless-network): Rename `NodeId`

### PR DESCRIPTION
**Pull request by teogeb**
Unify type naming: rename `NodeId` to `NodeID`.

### CODE REVIEW

File: packages/trackerless-network/src/logic/utils.ts [hunk 1]
```diff
@@ -40,3 +41,15 @@ export const hexToBinary = (hex: string): Uint8Array => {
...
+    return areEqualByteArrays(userId1, userId2)
+}
+
+const areEqualByteArrays = (arr1: Uint8Array, arr2: Uint8Array): boolean => {
+    if (arr1.byteLength === arr2.byteLength) {
+        return arr1.every((value, index) => value === arr2[index])
+    } else {
+        return false
+    }
+}
```
Comment (harbu) [inline packages/trackerless-network/src/logic/utils.ts]: There is a better way to compare with `Buffer`?

**Review by harbu — APPROVED** [review]
Maybe description and title should also include that the type of userId field changes in protobuf specification?

File: packages/trackerless-network/src/logic/utils.ts (hunk 1 above)
Comment (teogeb) [reply packages/trackerless-network/src/logic/utils.ts]: Yes, fixed that in the base PR (https://github.com/streamr-dev/network/pull/1787)

**Comment by teogeb** [comment]
> Maybe description and title should also include that the type of userId field changes in protobuf specification?

That change was mentioned in the base PR (https://github.com/streamr-dev/network/pull/1787). This PR is only about the `NodeID` type.

**Pull request merged**